### ISSUE TITLE
Removing ob-watcher.py "Shutdown" and "Reload"

### ISSUE THREAD

**Issue by btcspry** [chris-belcher/joinmarket#131]
Could we just remove the shutdown and reload buttons.  We could make it so that the reload occurs automatically and the shutdown can just be done by using `Ctrl+C` in the command line.  I think this would simplify things, and make it easier for the JoinMarket.io to merge up to there without having to make further changes.  What does everyone think?  I could submit a pull request for this if people are in agreement.

**Comment by CohibAA** [comment]
[Here](https://github.com/CohibAA/joinmarket-dev-cohibaa/blob/master/ob-watcher-modded.py) is the ob-watcher I use, but it requires

    <table class="sortable" in the tables 

in ob-watcher.py, and 

    <script src="sorttable.js"></script>

in the public html page, with a copy of [sorttable.js](https://github.com/CohibAA/joinmarket-dev-cohibaa/blob/master/sorttable.js) in the same directory as the public page.

I'm thinking maybe there should be 2 versions of ob-watcher... maybe the new one is ob-watcher-public.py, with wiki instructions for using ob-watcher.py on a local network only, or ob-watcher-public.py for a public orderbook page.  Or possibly, the main project is not aimed at public hosting of orderbook pages, and it should be left to users to make those modifications outside the main repo?

**Comment by btcspry** [comment]
I just don't see any benefits to having a shutdown button.  If we want ease for users, just have it update automatically.

**Comment by CohibAA** [comment]
Makes sense, I have never used the buttons, but I figured someone must have put them there to push them, lol.

**Comment by btcspry** [comment]
I just can't see them being useful.  You close that program the same way you'd close any other long-running CLI program.  And the reload button could easily be implemented so that the server refreshes it every 60 seconds or something.

**Comment by CohibAA** [comment]
Personally I run them locally in screen, or nohup, depending on what I want to be able to watch live versus reading logs.  I think I will implement auto-refresh on my public page though, because that does seem useful.

**Comment by chris-belcher** [comment]
Being able to make the bot say !orderbook can be useful, for example if there's a netsplit and you dont want to wait. Historically it was great for testing, there's less of that going on now but this is still nowhere near finished project.

I think I added shutdown because on windows ctrl+c does not kill it. Someone will have to try it as I don't remember.

Agreed with the periodic requesting of orderbook (although not as often as 60 seconds, we'll get kicked off the irc server, more like 30 minutes), it also fixes issue #50 

All this focus on the orderbook watcher reminds me of Parkinson's law of triviality.

> Parkinson observed and illustrated that a committee whose job was to approve plans for a nuclear power plant spent the majority of its time with pointless discussions on relatively trivial and unimportant but easy-to-grasp issues, such as what materials to use for the staff bike-shed, while neglecting the less-trivial proposed design of the nuclear power plant itself, which is far more important but also a far more difficult and complex task to criticise constructively.

ob-watcher is such a small part of the whole project so it seems somewhat wasteful to spend all this energy making it looks pretty.

**Comment by btcspry** [comment]
Ob-watcher is what runs on the JoinMarket.io website.  If I had no idea what GitHub was, and just wanted to use my bitcoin to ensure my anonymity, I would not think of [or even know how to] look on a GitHub wiki.  I would go to Google, type in JoinMarket, and then go to JoinMarket.io.  There, I would see a crude looking interface, and probably never try the project again.  If we make it something that's easy for people to see and understand, then you'll gain traction.  I'm a coder, and I completely agree there are better things to work on.  However, first impressions happen to be rather important.  I see great things happening with the project, but I think you (as the core developer here) need to get control of the domain, and make it look nicer (or work on ob-watcher so that the owner [who I've contacted to update the bootstrap layout] can easily put a nicer website that's open source and easily contributable on that site for beginners to see).

Going back to the real matter here, I think that getting rid of the button is important, and should be replaced with something automated (like the 30 minute suggestion).  In the Python program, you could make it so that it accepts commands, like if you type "quit" it quits the server and "reload" or "refresh" reloads the orderbook (but it also reloads it at a set interval, which could be specified in the config file or as a command line option).  These are easy fixes that make the project better. 

We should see the recent changes reflected on joinmarket.io in the near future.  The owner said he'd pull the changes right away.
